keyboards/ergoslab/rev1/rev1.h

```diff
@@ -0,0 +1,27 @@
+#ifndef REV1_H
```

Comment:
QMK uses the `#pragma once` include guard, instead of `#ifndef`/`#define`/`#endif`.

```suggestion
#pragma once
```